tools/cibadmin.c

```diff
@@ -194,8 +194,8 @@ static pcmk__cli_option_t long_options[] = {
     },
     {
         "-spacer-", no_argument, NULL, '-',
-        "\tValid values: nodes, resources, constraints, crm_config, "
-            "rsc_defaults, op_defaults, status",
+        "\tValid values: configuration, nodes, resources, constraints, "
+            "crm_config, rsc_defaults, op_defaults, acls, status",
```

Comment:
I'm guessing it would also work for options, fencing, tags, and alerts, but I haven't verified

Reply:
It uses known_paths[] from lib/cib/cib_utils.c so it would include cib (which would be redundant), all (same as cib/unspecified, so we could leave that off here), status (which users shouldn't touch so we don't need to mention here), and fencing-topology, but not tags or alerts (probably simply by oversight when those features were added). options isn't included either but I'm not sure whether that's intentional.

Interestingly it also allows "domains" which isn't in the schema, so it's dead code; that was an experiment at segregating nodes into failover domains (added in a773335 and removed in 800a12f)

Reply:
Thanks for the info! I should have looked for a list of valid values. I figured it might accept **any** unique CIB element. I see what it's doing now.

Ack re: "cib", "all", and "status". Technically they're valid options, but as you said there's no reason for an end user to use them.

I'll update with the full list minus the ones mentioned above and minus domains.

Should we get tags and alerts added and get domains removed?

Reply:
> Should we get tags and alerts added and get domains removed from `known_paths`?

Definitely